<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0" colspan="3">
     <widget class="QTextEdit" name="textEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="nextButton">
      <property name="text">
       <string>下一章</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="lastButton">
      <property name="text">
       <string>前一章</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="backButton">
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="changeDNButton">
      <property name="text">
       <string>夜间模式</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu1">
    <property name="title">
     <string>选择章节</string>
    </property>
    <addaction name="chapter_1"/>
    <addaction name="chapter_2"/>
    <addaction name="chapter_3"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>选择字体</string>
    </property>
    <addaction name="song"/>
    <addaction name="weiruan"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>字体大小</string>
    </property>
    <addaction name="size_9"/>
    <addaction name="size_12"/>
    <addaction name="size_15"/>
   </widget>
   <addaction name="menu1"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="chapter_1">
   <property name="text">
    <string>第一章</string>
   </property>
  </action>
  <action name="chapter_2">
   <property name="text">
    <string>第二章</string>
   </property>
  </action>
  <action name="song">
   <property name="text">
    <string>宋体</string>
   </property>
  </action>
  <action name="weiruan">
   <property name="text">
    <string>微软雅黑</string>
   </property>
  </action>
  <action name="size_9">
   <property name="text">
    <string>9</string>
   </property>
  </action>
  <action name="size_12">
   <property name="text">
    <string>12</string>
   </property>
  </action>
  <action name="size_15">
   <property name="text">
    <string>15</string>
   </property>
  </action>
  <action name="chapter_3">
   <property name="text">
    <string>第三章</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
